src/netius/test/clients/apn.py

```
@@ -0,0 +1,225 @@
+
+        self.protocol.connection_made(None)
+
+        # the making of the connection is what sends the notification,
+        # built from the values that were set before it
+        data = self.protocol.sent[0][0]
+        payload = json.loads(data[37:].decode("utf-8"))
+
+        self.assertEqual(payload["aps"]["alert"], "Hello World")
+
+    def test_notify(self):
+        if mock == None:
```

Ruff (0.16.2)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)